$NetBSD: patch-ac,v 1.1 2016/05/24 09:20:07 leot Exp $

Use curses(3) in NetBSD instead of ncursesw, and pkgsrc ncurses
(include/ncurses) on IRIX.

--- stfl_internals.h.orig
+++ stfl_internals.h
@@ -28,7 +28,15 @@
 #endif
 
 #include "stfl.h"
-#include <ncursesw/ncurses.h>
+#ifdef __NetBSD__
+# include <curses.h>
+# include <stdarg.h>
+#elif defined(__sgi)	/* pkgsrc ncurses, wide */
+# define NCURSES_WIDECHAR 1
+# include <ncurses/ncurses.h>
+#else
+# include <ncursesw/ncurses.h>
+#endif
 #include <pthread.h>
 
 struct stfl_widget_type;
